src/views/setup/MasterPassword.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+<b-container class="mt-3 align-content-center">
+    <b-row>
+      <h3 class="text-center pb-3">Welcome to the Setup from the COVID-Bouncer Admin</h3>
+    </b-row>
+  <b-row>
+    <form>
+        <b-form-group
+          label="1. Step: Setup the master password"
+          id="master-password"
+          class="input-styling">
+          <b-form-input
```

Comment:
There should be a check for minimal password length. I would suggest 8 chars.

Reply:
True